README.md

```diff
@@ -8,7 +8,7 @@ Client Application for Community Health West London
 
 `/`: Landing page (`<IndexView.jsx />`)
 
-`/about`: About section? (`<AboutView.jsx />`)
+`/about/us`: About section? (`<AboutView.jsx />`)
```

Comment:
I assume that we need to update this section with more routes?

Reply:
fixed